routes/v1.js

```diff
@@ -87,6 +88,7 @@ function addRoutes(app, peliasConfig) {
       postProc.renamePlacenames(),
       postProc.parseBoundingBox(),
       postProc.normalizeParentIds(),
+      postProc.assignLabels(require('pelias-labels')),
```

Comment:
I know that the purpose of this is so that we have a default in case a label generator isn't passed in, but how do clients actually take advantage of this?  They can certainly pass in their own function for label generation, but doesn't that mean that they have to modify source?  And doesn't that make their upgrade pretty difficult?  Wouldn't it make more sense to make these defaults configuration-driven?